debian/vyos-1x.postinst

```diff
@@ -53,6 +48,11 @@ if grep -q '^tacacs' /etc/passwd; then
     fi
 fi
 
+# Remove TACACS+ PAM default profile
+if [[ -e /usr/share/pam-configs/tacplus ]]; then
+    rm /usr/share/pam-configs/tacplus
```

Comment:
Deleting `/usr/share/pam-configs/tacplus` should not be required as the file is diverted in `debian/vyos-1x.preinst` script. We should not simply delete files but rather use debian tools and inform dpkg database about the changes.

Reply:
Unfortunately, this works only if a package installs something in place of a diverted file. In our case, there is no replacement and the file stays.

Also, diverted files will have another name, but `pam-auth-update` does not care about names and it will still use them.